packages/flutter/lib/src/material/tabs.dart

```diff
@@ -132,7 +132,10 @@ class Tab extends StatelessWidget implements PreferredSizeWidget {
   ///
   /// Only useful when used in combination with [icon], and either one of
   /// [text] or [child] is non-null.
-  final EdgeInsetsGeometry iconMargin;
+  ///
+  /// If [ThemeData.useMaterial3] is true, the default value is 2 pixels of bottom margin.
+  /// Otherwise, the default value is 10 pixels of bottom margin.
```

Comment:
```suggestion
  /// Defaults to 2 pixels of bottom margin. If [ThemeData.useMaterial3] is false,
  /// then defaults to 10 pixels of bottom margin.
```

The `useMaterial3` flag is `true` default.